assets/src/application/ui/calendars/dateDisplay/BiggieCalendarDate/index.tsx

```diff
@@ -0,0 +1,78 @@
+import classNames from 'classnames';
+import { format, isValid } from 'date-fns';
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import './style.css';
+import EspressoButton from '@appInputs/EspressoButton';
+
+export interface EditButtonProps {
+	tooltip: string;
+	tooltipPosition: string;
+}
+
+export enum Position {
+	LEFT = 'left',
+	RIGHT = 'right',
+}
+
+export interface BiggieCalendarDateProps {
+	date: Date;
+	editButton: EditButtonProps;
+	htmlClass: string;
+	headerText: string[] | string;
+	footerText: string[] | string;
+	onEdit: (event: React.MouseEvent<HTMLButtonElement>) => void;
+	position: Position;
+	showTime: boolean;
+}
+
+/**
+ * Displays a full calendar date, but REALLY BIG!!!
+ */
+export const BiggieCalendarDate: React.FC<BiggieCalendarDateProps> = ({
+	date,
+	htmlClass,
+	headerText,
+	footerText,
+	position = Position.LEFT,
+	onEdit = null,
+	editButton = {},
+	showTime = false,
+}) => {
+	if (!isValid(date)) {
+		return null;
+	}
+	const classes = classNames('ee-biggie-calendar-date-bg', htmlClass, {
```

Comment:
Although I do understand what is used for, maybe we should use just `className`/ `classNames`.
And for `classnames` package to have either the original name `classnames` or as we did in the past project smth like `cx`

Reply:
me no grok ?!?!?!

Reply:
I agree with @nicolad about using `classNames` as variable name instead of `classes`.